CMakeLists.txt

```diff
@@ -18,11 +18,28 @@ CALCULATE_LIBRARY_VERSIONS_FROM_LIBTOOL_TRIPLE(LIBWPEBACKEND_FDO 4 0 3)
 
 project(wpebackend-fdo VERSION "${PROJECT_VERSION}")
 
-set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fno-exceptions -fno-rtti")
-
+include(CheckCCompilerFlag)
+include(CheckCXXCompilerFlag)
 include(DistTargets)
 include(GNUInstallDirs)
 
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_C_STANDARD 99)
```

Comment:
Nowadays GCC defaults to C++14 and C17. I guess you're aware that you're regressing the language versions here.

Reply:
Note that for C++ before this patch `-std=c++11` was being passed to the compiler already:

```cmake
set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -fno-exceptions -fno-rtti")
```

On the C side, the only plain C code is `src/linux-dmabuf/linux-dmabuf-protocol.c` and the generated Wayland protocol code, none of which use C11 features, and that is the reason for choosing C99.